seqr/views/utils/terra_api_utils.py

```diff
@@ -65,17 +65,12 @@ def _get_social_access_token(user):
     return social.extra_data['access_token']
 
 
-def anvil_call(method, path, access_token, user=None, headers=None, root_url=None, **kwargs):
+def anvil_call(method, path, access_token, user=None, headers=None, root_url=None):
```

Comment:
you might also want to take out the `**kwargs` argument from `_user_anvil_call`, since it looks like its not used and its a bit confusing that you can pass through arbitrary kwargs to this function that might not be allowed

Reply:
okay